kubelet-to-gcm/monitor/kubelet/translate.go

```diff
@@ -617,7 +575,7 @@ func (t *Translator) getMonitoredResource(labels map[string]string) *v3.Monitore
 	resourceLabels["container_name"] = labels["container"]
```

Comment:
There should be `pod_name` and `namespace_name`, not id used here.

Reply:
Done.